pkgs/development/python-modules/torpy/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchFromGitHub
+, cryptography
+, pytest
+, requests
+}:
+
+buildPythonPackage rec {
+  pname = "torpy";
+  version = "1.1.6";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = "torpyorg";
+    repo = "torpy";
+    rev = "v${version}";
+    sha256 = "sha256-Ni7GcpkxzAMtP4wBOFsi4KnxK+nC0XCZR/2Z/eS/C+w=";
+  };
+
+  propagatedBuildInputs = [
+    cryptography
+    requests
+   ];
+
+  checkInputs = [
+    pytest
```

Comment:
```suggestion
    pytestCheckHook
```

Reply:
This makes the tests fail with `OSError: [Errno 101] Network is unreachable`

```
tests/integration/test_integration.py::test_clearnet_raw
-------------------------------- live log call ---------------------------------
[08:04:28] [MainThread      ] Loaded 10 authorities dir
[08:04:28] [MainThread      ] Loaded 141 fallbacks dir
[08:04:28] [MainThread      ] Downloading new consensus...
[08:04:28] [MainThread      ] There is no internal dir circuit yet. Creating it...
[08:04:28] [MainThread      ] There is no reasonable live consensus... use fallback dirs
[08:04:28] [MainThread      ] Connecting to guard node 5.189.169.190:8080... (Internal dir client)
[08:04:28] [MainThread      ] Connecting socket to 5.189.169.190:8080 relay...
[08:04:28] [MainThread      ] [Errno 101] Network is unreachable
[08:04:28] [MainThread      ] [ignored]
Traceback (most recent call last):
  File "/build/source/torpy/cell_socket.py", line 63, in connect
    self._socket.connect((self._router.ip, self._router.or_port))
  File "/nix/store/6v602p5l3c05iiq7jx8y0rjwiv2n8hhj-python3-3.9.13/lib/python3.9/ssl.py", line 1343, in connect
    self._real_connect(addr, False)
  File "/nix/store/6v602p5l3c05iiq7jx8y0rjwiv2n8hhj-python3-3.9.13/lib/python3.9/ssl.py", line 1330, in _real_connect
    super().connect(addr)
OSError: [Errno 101] Network is unreachable
```

Reply:
Please disable the network related tests. Right now they are not run at all.

```
running build_ext
----------------------------------------------------------------------
Ran 0 tests in 0.000s
OK
```

Reply:
I disabled all the tests in `tests/integration` as they all use the network.